lib/cli.js

```diff
@@ -47,6 +48,30 @@ const mkdir = promisify(fs.mkdir);
 const stat = promisify(fs.stat);
 const writeFile = promisify(fs.writeFile);
 
+/**
+ * Loads plugins with the specified names.
+ * @param {{ "import": (name: string) => Promise<any> }} importer An object with an `import` method called once for each plugin.
+ * @param {string[]} pluginNames The names of the plugins to be loaded, with or without the "eslint-plugin-" prefix.
+ * @returns {Promise<Record<string, Plugin>>} A mapping of plugin short names to implementations.
+ */
+async function loadPlugins(importer, pluginNames) {
+    const plugins = {};
+
+    await Promise.all(pluginNames.map(async pluginName => {
+
+        const shortName = naming.getShorthandName(pluginName, "eslint-plugin");
```

Comment:
[Style] Small thing: `shortName` isn't used until after the `if`. I think it'd make the code just a bit more clear to move the variable down a few lines. (Is this too nitpicky a comment?)